addons/mail/data/mail_channel_data.xml

```diff
@@ -4,15 +4,15 @@
 
         <record model="mail.channel" id="channel_all_employees">
             <field name="name">general</field>
-            <field name="group_ids" eval="[(4, ref('base.group_user'))]"/>
+            <field name="group_ids" eval="[Command.link(ref('base.group_user'))]"/>
```

Comment:
Maybe in the eval context we could have the methods at the toplevel instead of the class?

Reply:
I better like the xml code reads the same as the python one, here there is no ambiguity but a naked `unlink` command could be mistaken for the ORM's `unlink`.

Reply:
I don't think that's much of a concern, the XML loading eval context is very different than others, the ORM is not involved in it, and e.g. `ref` is already quite specific to it. Having to type `Command.` everytime is an unnecessary hassle.

Reply:
IMHO an explicit `Command.X` is better than simply `X`, which may be confusing.